Blockzilla/BrowserViewController.swift

```diff
@@ -1089,6 +1090,21 @@ extension BrowserViewController: OverlayViewDelegate {
         self.find(query, function: "find")
     }
     
+    func overlayView(_ overlayView: OverlayView, didAddToAutocomplete query: String) {
+        let autocompleteSource = CustomCompletionSource()
+        
+        switch autocompleteSource.add(suggestion: query) {
+        case .error(.duplicateDomain):
+            break
```

Comment:
On Android we handle this case by displaying this toast message: "URL already exists". We should likely do the same here.